### PR TITLE
MINOR: Update docs to say 2.2

### PR DESCRIPTION
**Pull request by mjsax**
*More detailed description of your change,
if necessary. The PR title and PR message become
the squashed commit message, so use a separate
comment to ping reviewers.*

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: docs/documentation.html [hunk 1]
```diff
@@ -26,8 +26,8 @@
...
+    <h3>Kafka 2.2 Documentation</h3>
+    Prior releases: <a href="/07/documentation.html">0.7.x</a>, <a href="/08/documentation.html">0.8.0</a>, <a href="/081/documentation.html">0.8.1.X</a>, <a href="/082/documentation.html">0.8.2.X</a>, <a href="/090/documentation.html">0.9.0.X</a>, <a href="/0100/documentation.html">0.10.0.X</a>, <a href="/0101/documentation.html">0.10.1.X</a>, <a href="/0102/documentation.html">0.10.2.X</a>, <a href="/0110/documentation.html">0.11.0.X</a>, <a href="/10/documentation.html">1.0.X</a>, <a href="/11/documentation.html">1.1.X</a>, <a href="/20/documentation.html">2.0.X</a>., <a href="/21/documentation.html">2.1.X</a>.
```
Comment (hachikuji) [inline docs/documentation.html]: We have an errant period in there after 2.0.x.

**Review by hachikuji — APPROVED** [review]
Left a trivial comment. LGTM once fixed.

**Review by bbejeck — APPROVED** [review]
LGTM

**Comment by bbejeck** [comment]
retest this please

**Comment by bbejeck** [comment]
checked out the branch and ran the tests locally so merging this now

**Comment by mjsax** [comment]
Was also cherry-picked to `2.2`.

**Pull request merged**